bin/xspec.sh

```diff
@@ -158,14 +159,34 @@ while echo "$1" | grep -- ^- >/dev/null 2>&1; do
                 usage "-t and -q are mutually exclusive"
                 exit 1
             fi
+            if test -n "$SCHEMATRON"; then
+                usage "-s and -t are mutually exclusive"
+                exit 1
+            fi
             XSLT=1;;
         # XQuery
         -q)
             if test -n "$XSLT"; then
                 usage "-t and -q are mutually exclusive"
                 exit 1
             fi
+            if test -n "$SCHEMATRON"; then
+                usage "-s and -q are mutually exclusive"
+                exit 1
+            fi
             XQUERY=1;;
+        # Schematron
+        -s)
+            if test -n "$XQUERY"; then
+                usage "-s and -q are mutually exclusive"
+                exit 1
+            fi
+            if test -n "$XSLT"; then
+                usage "-s and -t are mutually exclusive"
+                exit 1
+            fi
+            SCHEMATRON=1;
+            XSLT=1;;
```

Comment:
I had fixed this in xspec.bat but missed it in xspec.sh. Fixed now, and I added tests for the error messages about mutually exclusive parameters -s -t -q.